packages/js/src/components/WincherSEOPerformance.js

```diff
@@ -251,20 +371,95 @@ const TableContent = ( props ) => {
 		return <WincherNoKeyphraseSet />;
 	}
 
+	const historyLimitDate = moment().subtract( historyDaysLimit, "days" );
+
+	const periodOptions = WINCHER_PERIOD_OPTIONS.filter(
+		opt => moment( opt.value ).isSameOrAfter( historyLimitDate )
+	);
+
+	const defaultPeriod = orderBy( periodOptions, opt => opt.defaultIndex, "desc" )[ 0 ];
```

Comment:
Isn't this sorting unnecessary? Can't we just assume the constant is in the correct order? Like add a comment above it to future devs.

Reply:
Why do we rely on the future dev to be careful about it when we can already make the risk nil 😄